doc/cabal-package-description-file.rst

```diff
@@ -2676,43 +2676,31 @@ Starting with Cabal-2.2 it's possible to use common build info stanzas.
 
     TBW
 
-Source Repositories
-^^^^^^^^^^^^^^^^^^^
+
+.. _pkg-author-source:
+
+Package Source
```

Comment:
better "Package Source Metadata", "Package Source Code Origin", "Locate Package Source Code" or similar to better reveal the intent.

Reply:
Went with "Source Marker".